tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -699,4 +699,13 @@ test_i2c_s11059: s11059@6b {
 	compatible = "hamamatsu,s11059";
 	reg = <0x6b>;
 	integration-time = <546000>;
+
+ina3221: ina3221@6c {
+	compatible = "ti,ina3221";
+	reg = <0x6c>;
+	shunt-resistors = < 1000 1000 1000 >;
```

Comment:
@maxd-nordic - looks like spacing might be causing dtc compile issues. Can you try to remove the space after `<` and before `>`?

Reply:
There's a missing curly brace closing the previous node